manual/src/manual.tex

```diff
@@ -187,7 +186,6 @@
 \usepackage[strings,nohyphen]{underscore}
 
 %\makeatletter \def\@wrindex#1#2{\xdef \@indexfile{\csname #1@idxfile\endcsname}\@@wrindex#2||\\}\makeatother
-\def\th{^{\hbox{\scriptsize th}}}
```

Comment:
this uses `hbox` but the macros.hva version (old and new) use `mbox`. Does it make a difference?

Reply:
We should probably be using the latex macro `hbox` rather the tex one (`mbox`) but this doesn't really make a difference in html mode. Using `hbox` or `mbox` yields the same generated html.